### PR TITLE
Fixes for documentation system and minor errors

### PR DESCRIPTION
**Pull request by fsaad**
Fixes for documentation system and minor errors

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -62,32 +68,28 @@ plugins:
...
+                options:
                     # show_root_heading: true  # actually display anything at all...
-                    show_root_full_path: true  # display "diffrax.asdf" not just "asdf"
+                    show_symbol_type_heading: true
+                    show_symbol_type_toc: true
+                    show_root_full_path: false  # display "diffrax.asdf" not just "asdf"
                     show_if_no_docstring: false
                     show_signature_annotations: false
-                    show_source: false  # don't include source code
+                    show_source: true  # don't include source code
```
Comment (brianwa84) [inline mkdocs.yml:80]: s/don't//

File: docs/api/BayesianNeuralFieldMAP.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# BayesianNeuralFieldMAP
+
+::: bayesnf.spatiotemporal.BayesianNeuralFieldMAP
```
Comment (brianwa84) [inline docs/api/BayesianNeuralFieldMAP.md:3]: do you need the `.spatiotemporal` given the `__init__.py` change?

Comment (fsaad) [reply docs/api/BayesianNeuralFieldMAP.md:3]: The fully qualified path is needed
```
ERROR   -  Could not collect 'bayesnf.BayesianNeuralFieldEstimator'
```

File: mkdocs.yml (hunk 1 above)
Comment (fsaad) [reply mkdocs.yml]: Updated

**Review by brianwa84 — APPROVED** [review]

**Pull request merged**